src/main/java/com/gmail/goosius/siegewar/SiegeController.java

```diff
@@ -520,7 +522,7 @@ public static void startSiege(Block bannerBlock,
 		SiegeController.saveSiege(siege);
 
 		//Call event
-		Bukkit.getPluginManager().callEvent(new SiegeWarStartEvent(siege, townOfSiegeStarter, startMessage.defaultLocale()));
+		Bukkit.getPluginManager().callEvent(new SiegeWarStartEvent(siege, townOfSiegeStarter, siegeStarter, startMessage.defaultLocale()));

```

Comment:
I dont see the Event here being altered to accept a siegeStarter, one the presiege event.